packages/docusaurus-theme-classic/src/theme/DocPage/index.tsx

```diff
@@ -77,6 +77,7 @@ export default function DocPage(props: Props): JSX.Element {
       />
       <HtmlClassNameProvider
         className={clsx(
+          // TODO: it should be removed from here
```

Comment:
change looks fine

I don't think the top-level class should be removed because it's a stable one meant to be used for custom CSS targeting: one identity all the pages of the docs plugin, the other identity a specific page of the docs plugin

Reply:
But we already have `plugin-docs` that's applied to all docs plugin pages, right?

Reply:
oh yes

right, I think I kept those for temp easier retrocompat ;) we can keep the todo then